stdlib/src/builtin/simd.mojo

```diff
@@ -33,7 +33,7 @@ from utils._numerics import FPUtils
 from utils._numerics import isnan as _isnan
 from utils._numerics import nan as _nan
 from utils._visualizers import lldb_formatter_wrapping_type
-from utils import StaticTuple
+from utils import StaticTuple, StaticIntTuple
```

Comment:
**thoughts** I supposed this is here otherwise the compiler will complain that this does not exist. This is not the first time that I see this bug about the compiler not importing the builtins automatically. But it's really hard to reproduce.

Reply:
Is `StaticIntTuple`  supposed to be a builtin? it is defined in utils 
As I mentionned here it was exposed as a builtin from the imports in the int module: https://github.com/modularml/mojo/issues/2515#issuecomment-2094760396

Reply:
Then it seems you are importing it but not using it in simd.mojo. But maybe the PR is incomplete

Reply:
it was already used in simd.mojo, so I had to add the import here (it was missing but working "thanks" to the import in int.mojo)

Reply:
Yeah that's really not a good thing how builtin imports work right now. I'll open an issue.